pom.xml

```diff
@@ -33,6 +33,14 @@
     </developer>
   </developers>
 
+  <!--<repositories>-->
+    <!--<repository>-->
+      <!--<id>Apache-maven2</id>-->
+      <!--<name>Apache Maven2 Repository</name>-->
+      <!--<url>http://repo.maven.apache.org/maven2/</url>-->
+    <!--</repository>-->
+  <!--</repositories>-->
+
```

Comment:
Commenting this out for now, don't think it's needed (Maven2 Repository is used by default).

Reply:
Ya delete this